pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -519,15 +527,15 @@ access the same @tech{location}.
 
 Most definitions in Racket are in @deftech{modules}. In terms of evaluation,
 a module is essentially a prefix on a defined name, so that different
-modules can define the name. That is, a @deftech{module-level
+modules can define the same name. That is, a @deftech{module-level
 variable} is like a @tech{top-level variable} from the perspective of
 evaluation.
 
 One difference between a module and a top-level definition
 is that a module can be @deftech[#:key "declare"]{declared}
 without instantiating its module-level definitions.
-Evaluation of a @racket[require] @deftech{instantiates}
-(i.e., triggers the @deftech{instantiation} of) a declared
+Subsequently, evaluation of a @racket[require] @deftech{instantiates}
```

Comment:
-1 for adding "Subsequently, "

Reply:
You don't like that word. Is "Later," better?